climada/hazard/tc_tracks_synth.py

```diff
@@ -726,37 +748,60 @@ def _apply_decay_coeffs(track, v_rel, p_rel, land_geom, s_rel):
             continue
         if land_sea - sea_land == 1:
             continue
-        p_decay = _calc_decay_ps_value(track, p_landfall, land_sea - 1, s_rel)
-        p_decay = _decay_p_function(p_decay, p_rel[ss_scale_idx][1],
-                                    track.dist_since_lf[sea_land:land_sea].values)
-        # dont applay decay if it would decrease central pressure
-        p_decay[p_decay < 1] = track.central_pressure[sea_land:land_sea][p_decay < 1] / p_landfall
-        track.central_pressure[sea_land:land_sea] = p_landfall * p_decay
+        S = _calc_decay_ps_value(track, p_landfall, land_sea - 1, s_rel)
```

Comment:
I renamed to `S` because it is otherwise confusing.

Reply:
Could you use a more descriptive variable? Capital S is too short and non-descriptive.

Reply:
I used S because it is how it is call in the model, see e.g.
https://github.com/CLIMADA-project/climada_python/blob/41a110a0255b5a4fdf786973dd24bcbfbea82988/climada/hazard/tc_tracks_synth.py#L442
Should I rename it to e.g. `S_param`, to make it clear it's a parameter?

Reply:
I see, that is unfortunate. Then it is better to leave it as is.